website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: "docs"
+page_title: "Service Sync - Kubernetes"
+sidebar_current: "docs-platform-k8s-service-sync"
+description: |-
+  The services in Kubernetes and Consul can be automatically synced so that Kubernetes services are available to Consul agents and services in Consul can be available as first-class Kubernetes services.
+---
+
+# Syncing Kubernetes and Consul Services
+
+The services in Kubernetes and Consul can be automatically synced so that Kubernetes
+services are available to Consul agents and services in Consul can be available
+as first-class Kubernetes services. This functionality is provided by the
+[consul-k8s project](https://github.com/hashicorp/consul-k8s) and can be
+automatically installed and configured using the
+[Consul Helm chart](/docs/platform/k8s/helm.html).
+
+**Why sync Kubernetes services to Consul?** Kubernetes services synced to the
+Consul catalog enable Kubernetes services to be accessed by non-Kubernetes
+nodes that are part of the Consul cluster or by other distinct Kubernetes
```

Comment:
"...or by other distinct Kubernetes clusters."

Checking my understanding: these other Kubernetes clusters still need to be part of the Consul cluster for this to work, right? 

If so, this might be more clear:
"Kubernetes services synced to the Consul catalog enable Kubernetes services to be accessed by other Consul-registered services, whether they are non-Kubernetes nodes or part of a different Kubernetes cluster."